dev/CommonStyles/ToggleButton_themeresources.xaml

```diff
@@ -180,6 +180,8 @@
         </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
 
+    <contract7Present:BackgroundSizing x:Key="ToggleButtonCheckedStateBackgroundSizing">OuterBorderEdge</contract7Present:BackgroundSizing>
```

Comment:
You should put this in the theme resources dictionaries so it is customizable at the app level. @StephenLPeters as FYI

Reply:
@ranjeshj, does it mean we need to put all the resources we have in theme resource dictionaries?

Reply:
Yes, if you want an app author to be able to customize a resource from anywhere it needs to be defined in the ThemeDictionaries (in triplicate) and referenced by {ThemeResource}